packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -480,22 +454,15 @@ export interface IGCRuntimeOptions {
  */
 export interface IGarbageCollectorConfigs {
 	/**
-	 * Tracks if GC is enabled for this document. GC may not be enabled for old documents created pre-GC.
+	 * Tracks if GC is allowed for this document. GC may not be allowed for old documents created pre-GC.
 	 */
-	readonly gcEnabled: boolean;
+	readonly gcAllowed: boolean;
 	/**
-	 * Tracks if sweep phase is allowed for this document. This is specified during document creation and doesn't change
-	 * throughout its lifetime.
+	 * Tracks if sweep phase is enabled to run in this session or not.
 	 */
 	readonly sweepEnabled: boolean;
```

Comment:
We also want sweepAllowed returned as part of the configs, for logging, even though it isn't used directly.